Jenkinsfile

```diff
@@ -1,207 +1,301 @@
 deployBranches = ['master']
+isRelease = deployBranches.contains(env.BRANCH_NAME)
 
 configurationTypes = [
-    ['Windows_10_85', 'chrome'],
-    ['Windows_10_81', 'firefox'],
-    ['Windows_7_80', 'chrome'],
-    ['Windows_7_78', 'firefox'],
-    ['OSX_Mojave_84', 'chrome'],
-    ['OSX_Mojave_12', 'safari'],
-    ['iPhone11_13', 'ios'],
-    ['iPhone8_12', 'ios'],
+  ['Windows_10_85', 'chrome'],
+  ['Windows_10_81', 'firefox'],
+  ['Windows_7_80', 'chrome'],
+  ['Windows_7_78', 'firefox'],
+  ['OSX_Mojave_84', 'chrome'],
+  ['OSX_Mojave_12', 'safari'],
+  ['iPhone11_13', 'ios'],
+  ['iPhone8_12', 'ios'],
 ]
 
-cron_schedule = deployBranches.contains(BRANCH_NAME) ? '0 2 * * *' : ''
+cron_schedule = isRelease ? '0 2 * * *' : ''
 
-def docker_registry = '979633842206.dkr.ecr.eu-west-1.amazonaws.com'
-def docker_registry_url = "https://${docker_registry}"
-def ecr_credential = 'ecr:eu-west-1:cita-devops'
+docker_registry = '979633842206.dkr.ecr.eu-west-1.amazonaws.com'
+docker_registry_url = "https://${docker_registry}"
+ecr_credential = 'ecr:eu-west-1:cita-devops'
 
-def images = [:]
+images = [:]
 
-pipeline {
-    triggers { cron(cron_schedule) }
-    agent {
-        label 'docker && awsaccess'
-    }
-    environment {
-        DOCKER_TAG = "${env.BRANCH_NAME}_${getSha()}"
-        // Using the commit SHA would mean that every build gets a different tag and this busts the cache between PR builds.
-        CA_STYLEGUIDE_VERSION_TAG = "${env.BRANCH_NAME}"
-        BUILD_STAGE = ''
-    }
-    parameters {
-        string(name: 'slackChannel', defaultValue: '#new_platform_builds',
-            description: 'channel to print messages')
-        string(name: 'slackCredentialsId', defaultValue: 'slack-plugin',
-            description: 'id of slack credentials used when posting a message')
+global_environment_variables = []
+
+// Pipeline definition begins here
+
+node('docker && awsaccess') {
+  try {
+    step([
```

Comment:
no need to use the generic step syntax, this is already exposed as `cleanWs()`.
`cleanWs()` uses deferred cleanup by default so no need to add it to `try ..finally`

Reply:
I changed it because I got a no such method error

https://jenkins.devops.citizensadvice.org.uk/view/Design%20System/job/design-system/view/change-requests/job/PR-531/2/console

Reply:
typo - you used `cleanWS()`, it should be `cleanWs()`

Reply:
dammit, that list hid the fact it wasn't alphabetically ordered in plain sight